### PR TITLE
[Integration] preparing for deployment in PyPI

### PR DESCRIPTION
**Pull request by TaykYoku**
The purpose of this PR is to change the code structure and add the necessary files to prepare the project for deployment in PyPI. More details here https://packaging.python.org/tutorials/packaging-projects/.

 - dcb9aca, 54150a0 - add the necessary files and structure to create the package
 - 8ddf5a0, 3194efa - align commands with [DIRAC developing instruction](https://dirac.readthedocs.io/en/integration/DeveloperGuide/AddingNewComponents/DevelopingCommands/index.html) and add their to setup
 - aling with python3
 
I think this changes related to #72.

### CODE REVIEW

File: src/COMDIRAC/Interfaces/Utilities/DConfigCache.py [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
 import stat
 import time
-import cPickle
 import re
 
-from DIRAC.Core.Base import Script
+try:
+  import cPickle as pickle  # python 2
+except ImportError:
+  pass
```
Comment (pigay) [inline src/COMDIRAC/Interfaces/Utilities/DConfigCache.py]: shouldn't python3 import pickle?

File: src/COMDIRAC/Interfaces/scripts/dgetenv.py [hunk 2]
```diff
@@ -1,18 +1,22 @@
 #! /usr/bin/env python
 
 """
-print DCommands session environment variables
+print(DCommands session environment variables)
```
Comment (pigay) [inline src/COMDIRAC/Interfaces/scripts/dgetenv.py]: improper parentheses in a comment ;)

File: src/COMDIRAC/Interfaces/scripts/dls.py [hunk 3]
```diff
@@ -112,7 +118,7 @@ def getHuman( self ):
 
   class ReplicaDirectoryListing( DirectoryListing ):
     def addFileWithReplicas( self, name, fileDict, numericid, replicas ):
-      """ Pretty print of the file ls output with replica info
+      """ Pretty print(of the file ls output with replica info)
```
Comment (pigay) [inline src/COMDIRAC/Interfaces/scripts/dls.py]: Please, restore comment

File: src/COMDIRAC/Interfaces/scripts/dls.py [hunk 4]
```diff
@@ -166,18 +172,18 @@ def printListing( self, reverse, timeorder, sizeorder, humanread ):
...
+        print(str( e[ 0 ] ),)
+        print(str( e[ 1 ] ).rjust( wList[ 1 ] ),)
+        print(str( e[ 2 ] ).ljust( wList[ 2 ] ),)
+        print(str( e[ 3 ] ).ljust( wList[ 3 ] ),)
+        print(str( size ).rjust( wList[ 4 ] ),)
+        print(str( e[ 5 ] ).rjust( wList[ 5 ] ),)
+        print(str( e[ 6 ] ))
   
-        # print replicas if present
+        # print(replicas if present)
```
Comment (pigay) [inline src/COMDIRAC/Interfaces/scripts/dls.py]: Please, restore comment

File: src/COMDIRAC/Interfaces/scripts/dpwd.py [hunk 5]
```diff
@@ -1,11 +1,17 @@
 #! /usr/bin/env python
 
 """
-print DCommands working directory
+print(DCommands working directory)
```
Comment (pigay) [inline src/COMDIRAC/Interfaces/scripts/dpwd.py]: PLease, restore comment

File: src/COMDIRAC/Interfaces/scripts/dsize.py [hunk 6]
```diff
@@ -1,27 +1,33 @@
 #! /usr/bin/env python
 
 """
-print FileCatalog file or directory disk usage
+print(FileCatalog file or directory disk usage)
```
Comment (pigay) [inline src/COMDIRAC/Interfaces/scripts/dsize.py]: Please, restore comments

**Review by pigay — CHANGES_REQUESTED** [review]
Very nice!

Apart from the py3 pickle import missing, a few comments where `print something` was abusively  changed into `print (something)` ;)

Pierre

File: src/COMDIRAC/Interfaces/Utilities/DConfigCache.py (hunk 1 above)
Comment (TaykYoku) [reply src/COMDIRAC/Interfaces/Utilities/DConfigCache.py]: Yes, as far as I know) fixed

File: src/COMDIRAC/Interfaces/scripts/dgetenv.py (hunk 2 above)
Comment (TaykYoku) [reply src/COMDIRAC/Interfaces/scripts/dgetenv.py]: Here I clearly overdid it.. fixed

File: src/COMDIRAC/Interfaces/scripts/dls.py (hunk 3 above)
Comment (TaykYoku) [reply src/COMDIRAC/Interfaces/scripts/dls.py]: fixed

File: src/COMDIRAC/Interfaces/scripts/dls.py (hunk 4 above)
Comment (TaykYoku) [reply src/COMDIRAC/Interfaces/scripts/dls.py]: fixed

File: src/COMDIRAC/Interfaces/scripts/dpwd.py (hunk 5 above)
Comment (TaykYoku) [reply src/COMDIRAC/Interfaces/scripts/dpwd.py]: fixed

File: src/COMDIRAC/Interfaces/scripts/dsize.py (hunk 6 above)
Comment (TaykYoku) [reply src/COMDIRAC/Interfaces/scripts/dsize.py]: fixed

**Review by pigay — APPROVED** [review]
Thanks!

**Pull request merged**